src/tools/dashboard/src/books.rs

```diff
@@ -296,17 +291,15 @@ fn extract(par_from: &Path, par_to: &Path, config_paths: &mut HashSet<PathBuf>)
     for mut example in examples.0 {
         apply_diff(par_to, &mut example, config_paths);
         example.config.edition = Some(example.config.edition.unwrap_or(Edition::Edition2021));
-        example.code = pub_main(
-            rustdoc::doctest::make_test(
-                &example.code,
-                None,
-                false,
-                &Default::default(),
-                example.config.edition.unwrap(),
-                None,
-            )
-            .0,
-        );
+        example.code = rustdoc::doctest::make_test(
+            &example.code,
+            None,
+            false,
+            &Default::default(),
+            example.config.edition.unwrap_or(Edition::Edition2018),
```

Comment:
The default edition was set to 2021 in #652, so this should be:
```suggestion
            example.config.edition.unwrap(),
```